spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/domain/SpringBootAppKindReader.java

```diff
@@ -43,8 +50,13 @@
 				springBootAppKindList.add(springBootAppKind);
 			}
 		}
+		catch (JsonMappingException e) {
+			logger.error("Can't parse Package's manifest YAML = " + manifest);
```

Comment:
@markpollack We may not want to display the entire manifest here as this could have some credentials as well. What do you think?